### PR TITLE
#4 - About Page

### PR DESCRIPTION
**Pull request by qvuqvu**

### CODE REVIEW

File: src/pages/about.jsx [hunk 1]
```diff
@@ -1,5 +1,82 @@
...
+        <p className="laptop:text-9 laptop:pt-16 laptop:pb-20 laptop:px-[450px]  text-h2 py-8 px-9 ">
+          A brand built on the love of craftmanship, quality and outstanding
+          customer service
+        </p>
+      </div>
+      {/* Features sections */}
+      <div className="grid grid-cols-1 laptop:grid-cols-2 mx-auto">
+        {/* Feature 1 */}
+        <div>
+          <div className="px-6 mt-12 sm:mb-9 laptop:px-28  laptop:pt-20 ">
```
Comment (vuhuy172dh) [inline src/pages/about.jsx]: remove "sm" in "sm:mb-9"

File: src/pages/about.jsx [hunk 2]
```diff
@@ -1,5 +1,82 @@
...
+            <div className="text-h4 laptop:text-6">
+              From a studio in London to a global brand with over 400 outlets
+            </div>
+            <p className="font-satoshi text-3.5 mt-3 laptop:text-4">
```
Comment (vuhuy172dh) [inline src/pages/about.jsx]: replace "font-satoshi text-3.5 laptop:text-4" with "text-body-sm laptop:text-body-md". I have declared 3 more text styles that applies font satoshi in index.css

File: src/pages/about.jsx [hunk 3]
```diff
@@ -1,5 +1,82 @@
...
+        <div>
+          <img
+            src={feature2}
+            alt="feature2"
+            className=" w-full  object-cover"
+          />
+        </div>
+        {/* Feature 3 */}
+        <div>
+          <img src={feature3} alt="feature1" className="  h-full object-cover" />
```
Comment (vuhuy172dh) [inline src/pages/about.jsx]: replace "h-full" with "w-full"

File: src/pages/about.jsx [hunk 4]
```diff
@@ -1,5 +1,82 @@
...
+        </div>
+        {/* Feature 4*/}
+        <div>
+          <div className="px-6 mt-12 mb-9 laptop:px-28  laptop:pt-20" >
+            <div className="text-h4 laptop:text-6 ">
+              Our service isn’t just personal, it’s actually hyper personally
+              exquisite
+            </div>
+            <p className="font-satoshi text-3.5 mt-3 laptop:text-4">
```
Comment (vuhuy172dh) [inline src/pages/about.jsx]: replace "font-satoshi text-3.5 laptop:text-4" with "text-body-sm laptop:text-body-md".

File: src/pages/about.jsx [hunk 5]
```diff
@@ -1,5 +1,82 @@
...
+              When we started Avion, the idea was simple. Make high quality
+              furniture affordable and available for the mass market. <br />{' '}
+              <br />
+              Handmade, and lovingly crafted furniture and homeware is what we
+              live, breathe and design so our Chelsea boutique become the hotbed
+              for the London interior design community.
+            </p>
+            <button className="laptop:w-2/5 laptop:mt-40 mobile:mb-16 mt-16 w-full h-14 bg-light_grey">
+              Get in touch
+            </button>
```
Comment (vuhuy172dh) [inline src/pages/about.jsx]: replace button by Button component. You should read button.jsx in components folder to use. You shouldn't edit button.jsx, just apply it to your file

File: src/pages/about.jsx [hunk 6]
```diff
@@ -1,5 +1,82 @@
...
+            <p className="font-satoshi text-3.5 mt-3 laptop:text-4">
+              When we started Avion, the idea was simple. Make high quality
+              furniture affordable and available for the mass market. <br />
+              <br /> Handmade, and lovingly crafted furniture and homeware is
+              what we live, breathe and design so our Chelsea boutique become
+              the hotbed for the London interior design community.
+            </p>
+            <button className="laptop:w-2/5 laptop:mt-48 mt-16 mb-16 w-full h-14 bg-light_grey">
+              Get in touch
+            </button>
```
Comment (vuhuy172dh) [inline src/pages/about.jsx]: replace this by Button component.

File: src/pages/about.jsx (hunk 1 above)
Comment (qvuqvu) [reply src/pages/about.jsx]: Done

File: src/pages/about.jsx (hunk 2 above)
Comment (qvuqvu) [reply src/pages/about.jsx]: Done

File: src/pages/about.jsx (hunk 4 above)
Comment (qvuqvu) [reply src/pages/about.jsx]: Done

File: src/pages/about.jsx (hunk 5 above)
Comment (qvuqvu) [reply src/pages/about.jsx]: Done

File: src/pages/about.jsx (hunk 3 above)
Comment (qvuqvu) [reply src/pages/about.jsx]: I've tried this but the images could not fit the div

Comment (qvuqvu) [reply src/pages/about.jsx]: > replace "h-full" with "w-full"

I've tried this but the images could not fit the div

**Comment by vuhuy172dh** [comment]
Merged!!!!

**Pull request merged**